tasks/install-RedHat.yml

```diff
@@ -2,5 +2,5 @@
 - name: Install MLNX_OFED packages for Red Hat OS family
   ansible.builtin.package:
     name:
-      - kmod-mlnx-ofa_kernel
+      - 	mlnx-ofa_kernel-devel
```

Comment:
You can't remove `kmod-mlnx-ofa_kernel`. Please install both packages.

Reply:
oui j'ai oublié de le remettre  fixed